dbt/models/churnado_subscriptions.sql

```diff
@@ -0,0 +1,41 @@
+select
+  s.id as subscription_id
+  , s.customer_id
+  , s.plan_id
+  , s.simplified_plan_id
+  , s.canceled_at
+  , s.created_at
+  , s.quantity
+  , s.trial_start_at
+  , s.trial_end_at
+  , u.id as user_id
+  , datediff(day, s.trial_start_at, s.trial_end_at) as trial_length
```

Comment:
I wasn't sure how are we handling trials. If we're creating new subscriptions for each trial we'll need to remove this to avoid any confusion.

Reply:
Yeah, I think we can exclude this 😄